index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Angular Application</title>
+</head>
+<body>
+
+    <div ng-app="app">
+        <div ng-controller="MainController">
+            Hello, {{ name }}!
+        </div>
+    </div>
+
+    <script src="js/angular.js"></script>
+    <script src="js/app/app.js"></script>
+    <script src="js/app/controllers/MainController.js"></script>
```

Comment:
Yeah you're missing the export of the Angular module, app.js should only include the angular.module('app', []); and MainController.js has all of this:

```js
function MainController($scope) {
  $scope.name = 'PUT YOUR NAME HERE!';
}

angular
  .module('app')
  .controller('MainController', MainController);
```